test/cmd/apply.sh

```diff
@@ -444,6 +444,16 @@ run_kubectl_server_side_apply_tests() {
 }
 __EOF__
 
+  # Ensure the API server has recognized and started serving the associated CR API
+  local tries=5
+  for i in $(seq 1 $tries); do
+      if kubectl "${kube_flags_with_token[@]}" api-resources --api-group 'mygroup.example.com' -oname | grep -q 'resources.mygroup.example.com'; then
```

Comment:
piping to `grep -q` breaks things, see https://github.com/kubernetes/kubernetes/pull/57805